GUI/DeusEx/Classes/HUDMedBotAddAugsScreen.uc

```diff
@@ -8,6 +8,25 @@ class DXRHUDMedBotAddAugsScreen extends HUDMedBotAddAugsScreen;
 
 var PersonaActionButtonWindow btnRemove;
 
+function CreateControls()
+{
+	CreateNavBarWindow();
+	CreateClientBorderWindow();
+	CreateClientWindow();
+
+	CreateTitleWindow(9, 5, AugmentationsTitleText);
+	CreateInfoWindow();
+	CreateButtons();
+	CreateAugmentationLabels();
+	CreateAugmentationHighlights();
+	CreateAugmentationButtons();
+	CreateOverlaysWindow();
+	CreateBodyWindow();
+	CreateAugsLabel();
+	CreateAugCanList();
+	// CreateMedbotLabel();
```

Comment:
if we're only copying the CreateControls function to remove CreateMedbotLabel, maybe we should just override CreateMedbotLabel with an empty function, and then SetMedicalBot could call `Super.CreateMedbotLabel()`

either way seems fine to me though

Reply:
I like that more, yeah. It's simpler and is in line with the other changes.